templates/atom/apps/qubit/config/factories.yml

```diff
@@ -4,14 +4,14 @@
 
 prod:
   storage:
-{% if atom_app_cache_engine == "sfMemcacheCache" %}
+{% if atom_app_cache_engine == "sfMemcacheCache" or atom_app_cache_engine == "sfAPCCache" %}
```

Comment:
I think this condition will always be true, so is it needed?

Reply:
I have the same doubt. Tried to keep the PR change as specific as possible to prevent any unexpected issues (as the symfony docs mention quite a few built-in cache classes apart from the 2 we currently use)